pkg/utils/perm_windows.go

```diff
@@ -63,7 +63,7 @@ func CurrentFileOwner() (FileOwner, error) {
 
 // HasStrictExecPerms ensures that the path is executable by the owner and that the owner of the file
 // is the same as the UID or root.
-func HasStrictExecPerms(path string, uid int) error {
+func HasStrictExecPerms(path string, uid int, _ bool) error {
```

Comment:
We should fix this, an arbitrary user being able to control capabilities.yml now has the ability override agent configurations from Fleet (preventing upgrades also would not be very nice).

Is there a way to scope the capability to only work in containers instead? Then you would have a way to avoid impact on endpoint use cases for example and avoid windows completely.